tests/test_curv_backends_interface.py

```diff
@@ -361,3 +408,92 @@ def test_full_ef_reg_curvlinops_against_backpack_full(reg_Xy, model):
     loss_f, H_ggn = backend.full(X, y)
     assert torch.allclose(loss, loss_f)
     assert torch.allclose(fggn, H_ggn, atol=0.0001)
+
+
+@pytest.mark.parametrize(
+    "backend_cls",
+    [
+        CurvlinopsGGN,
+        CurvlinopsEF,
+        CurvlinopsHessian,
+        AsdlGGN,
+        AsdlEF,
+        AsdlHessian,
+        AsdfghjklGGN,
+        AsdfghjklEF,
+        AsdfghjklHessian,
+        BackPackGGN,
+        BackPackEF,
+    ],
+)
+@pytest.mark.parametrize("method", ["full", "kron", "diag"])
+@pytest.mark.parametrize("logit_class_dim", [-1, 1000])
+def test_logit_class_dim_class(backend_cls, method, logit_class_dim, model, class_Xy):
```

Comment:
I think this has to be tested for actual equivalence of the computation, i.e. `pytest.mark.parametrize` the tests of the backends with `logit_class_dim`.